.github/workflows/deploy_dev.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy
+
+on:
+  workflow_dispatch:
+
+env:
+  NOTIFY_MATRIX: false
+  NOTIFY_TELEGRAM: true
+  GITHUB_WORKFLOW_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 🛎 Checkout
+        id: checkout
+        uses: actions/checkout@v3
+
+      - uses: ./.github/workflows/rust-install
+
+      - name: ⚙ Run verifier
+        id: verify
+        run: make verifier
+
+      - name: ⚙ Run collector
+        id: collect
+        run: |
+          cargo run --release -- -c=config_dev.toml collect
+          exit_code=$?
+          if [ $exit_code -eq 12 ]
+          then
+            echo "::set-output name=redeploy::true"
+            exit 0
+          fi
+          echo "::set-output name=redeploy::false"
+          exit $exit_code
+        shell: bash {0}
+
+      - uses: ./.github/workflows/deploy
```

Comment:
looks like it is the same action as for the [main deployment](https://github.com/nova-wallet/metadata-portal/blob/master/.github/workflows/deploy.yml#L42)
Isn't this version going to overlay the main version?

Reply:
Right, the action is the same. It just builds the NPM package and updates the gh-pages. NPM is the same for dev and prod builds.
Thank you for noticing it. I missed it and it may cause a big problem. Lucky that deploy isn't environment dependent.